x-pack/platform/plugins/shared/osquery/common/api/action_results/get_action_results_route.ts

```diff
@@ -16,7 +16,6 @@ export const getActionResultsRequestQuerySchema = t.type({
   sort: t.union([t.string, t.undefined]),
   sortOrder: t.union([t.literal(Direction.asc), t.literal(Direction.desc), t.undefined]),
   kuery: t.union([t.string, t.undefined]),
-  agentIds: t.union([t.string, t.undefined]),
```

Comment:
Question: Isn't the purpose of `agentIds` param to allow users of this public api to fetch results of agent specific queries only? If so, shouldn't we leave this param be and modify the logic only for when it's empty and at the same time send empty agent list from UI?